pandas/tseries/offsets.py

```diff
@@ -112,6 +112,41 @@ def wrapper(self, other):
     return wrapper
 
 
+def apply_index_wraps(func):
```

Comment:
what is going on here?

Reply:
Been awhile. I don't remember writing this code. I'll guess an artifact of my rebase.